apps/explorer/src/components/owned-objects/views/DynamicFieldsCard.tsx

```diff
@@ -85,6 +82,10 @@ export function DynamicFieldsCard({ id }: { id: string }) {
                                             <UnderlyingObjectCard
                                                 parentId={id}
                                                 name={result.name}
+                                                isDynamicObjectType={
```

Comment:
why not just pass the dynamicField type directly and move this check to the component?